test/test.js

```diff
@@ -387,10 +385,14 @@ describe( 'Utility Methods: ', function() {
 		process.argv[2] = '-c'
 		process.argv[3] = '.stylintrc'
 		var testMethod = app.setConfig( '.stylintrc' )
-		var testConfig = JSON.parse( stripJsonComments( fs.readFileSync( process.cwd() + '/.stylintrc', 'utf-8' ) ) )
+		var testConfig = app.setConfig( process.cwd() + '/.stylintrc' )
+
+		it( 'return complete config when not complete config is passed in', function() {
+			assert.ok( testConfig.reporter )
+		} )
 
 		it( 'update config state if passed a valid path', function() {
-			assert.deepEqual( testMethod, testConfig )
+			assert.equal( Object.keys( testConfig ).length, Object.keys( testMethod ).length )
```

Comment:
Why are we changing this test? If we must change this, we should still check for deep equality between two objects since doing key comparisons can lead to false positives.

Reply:
Trying to remember this one.. 😣 

I think I changed it since the important part is that they should contain the same amount of options now, not the exact same values since they may vary. I'll try to dig out my old notes.